manifests/osfamily/redhat.pp

```diff
@@ -6,7 +6,7 @@
   $pa_collection = getvar('::puppet_agent::collection')
 
   if $::operatingsystem == 'Fedora' {
-    if $pa_collection == 'PC1' {
+    if $pa_collection == 'PC1' or $pa_collection == 'puppet5' {
```

Comment:
I don't believe this will successfully exclude puppet6 -- internal builds are still shipping to PC1 everywhere. puppet-agent does use a 5.99.z version in master now, though

Reply:
I don't think that should matter. I'd have to go back and look, but pretty sure `$pa_collection` will figure out the right thing. Even though things land under PC1 on builds, that gets rearranged and whatnot later.

Reply:
Ok, that sounds good to me!